cases/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 __all__ = ['CONF']
 
-CONF = list(yaml.load_all(open("./config.yml")))[0]
+CONF = list(yaml.load_all(open("/home/ramakasturinarra/automation/cockpit-auto/config.yml")))[0]
```

Comment:
don't hard code local path into constants, consider make it dynamic or put it in separate config file